aif360/algorithms/inprocessing/gerryfair/classifier_history.py

```diff
@@ -0,0 +1,14 @@
+class ClassifierHistory(object):
```

Comment:
I don't see how this is more than just a `list`. Is it necessary?

Reply:
its two lists, i didn't add this piece of the code but i guess its nice having the classifier objects and the predictions bound together in one object. but yes could have been accomplished by a dictionary of lists. I'm going to leave as is this doesn't seem like a huge deal.